### PR TITLE
My Rock Paper Scissors Project

### PR DESCRIPTION
**Pull request by mudimurtala**
I've worked on the steps except for the last one, I decided to push what I've done so far while still working on the remaining part.

### CODE REVIEW

**Comment by mudimurtala** [comment]
I've completed the project and I'm ready for review.

File: script.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+let computerScore = 0;
+
+function getComputerChoice() {
+    const result = Math.floor(Math.random() * 3) + 1;
+
+    if (result === 1)
+        {
+            return "rock";
+        }
+    else if (result === 2)
```
Comment (FlatBallFlyer) [inline script.js]: From a code formatting perspective, most developers use 
```
if (result == 1) {
    return "rock"
}
```
It's 1 line shorter, and it makes the close } line up properly. This is such a common practice that VSCode will reformat your code. On linux you can use the short-cut Ctrl+Shift+I to reformat the code.

File: script.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+
+
+function playRound(humanChoice, computerChoice) {
+
+    console.log(`Human: ${humanChoice}, Computer: ${computerChoice}`);
+    if ((humanChoice === "rock" && computerChoice === "rock") ||
+        (humanChoice === "paper" && computerChoice === "paper") ||
+        (humanChoice === "scissors" && computerChoice === "scissors")) {
+            console.log("No winner for this round");
```
Comment (FlatBallFlyer) [inline script.js:31]: Is there an easier way to do this?

File: script.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    {
+        option = prompt("Type in any of these: rock, paper, or scissors:").toLowerCase();
+
+        if (option === "rock" || option === "paper" || option === "scissors")
+            {
+            return option;
+            }
+        else
+            {
+            alert("You choose a wrong option! Choose again");
```
Comment (FlatBallFlyer) [inline script.js:23]: Error handling! - Extra points for doing this!

**Review by FlatBallFlyer — APPROVED** [review]
It works, which is the most important part. I've left some feedback below.

**Comment by mudimurtala** [comment]
I have made some changes to my code:
1- I improved on `getComputerChoice()` function by using an array to fetch the choices rather than the return statement I used earlier.
2- I followed the normal style convention concerning the placement of curly braces `{}`.
3- I also replace this entire code:
````
if ((humanChoice === "rock" && computerChoice === "rock") ||
 (humanChoice === "paper" && computerChoice === "paper") ||
 (humanChoice === "scissors" && computerChoice === "scissors"))
````
with this shorter one: `if (humanChoice === computerChoice)`.

**Pull request merged**